Formula/molten-vk.rb

```diff
@@ -0,0 +1,137 @@
+    # External/glslang/known_good.json
+    url "https://github.com/KhronosGroup/SPIRV-Tools.git",
+        :revision => "26c1b8878315a7a5c188df45e0bc236bb222b698"
+  end
+
+  resource "spirv-headers" do
+    # External/glslang/known_good.json
+    url "https://github.com/KhronosGroup/SPIRV-Headers.git",
+        :revision => "2434b89345a50c018c84f42a310b0fad4f3fd94f"
+  end
+
+  resource "vulkan-tools" do
+    # ExternalRevisions/Vulkan-Tools_repo_revision
+    url "https://github.com/KhronosGroup/Vulkan-Tools.git",
+        :revision => "2abb69904b9ad017d39d3da1e7fc3dec1a584cd8"
+  end
+
+  def install
+    (buildpath/"External/cereal").install resource("cereal")
+    (buildpath/"External/Vulkan-Headers").install resource("vulkan-headers")
+    (buildpath/"External/Vulkan-Portability").install resource("vulkan-portability")
+    (buildpath/"External/SPIRV-Cross").install resource("spirv-cross")
+    (buildpath/"External/glslang").install resource("glslang")
+    (buildpath/"External/glslang/External/spirv-tools").install resource("spirv-tools")
+    (buildpath/"External/glslang/External/spirv-tools/external/spirv-headers").install resource("spirv-headers")
+    (buildpath/"External/Vulkan-Tools").install resource("vulkan-tools")
+
+    mkdir "External/glslang/External/SPIRV-Tools/build" do
+      # Required due to files being generated during build.
+      system "cmake", "..", *std_cmake_args
+      system "make"
+    end
+
+    xcodebuild "-project", "ExternalDependencies.xcodeproj",
+               "-scheme", "ExternalDependencies-macOS",
+               "-derivedDataPath", "External/build",
+               "SYMROOT=External/build", "OBJROOT=External/build",
+               "build"
+
+    xcodebuild "-project", "MoltenVKPackaging.xcodeproj",
+               "-scheme", "MoltenVK Package (macOS only)",
+               "SYMROOT=#{buildpath}/build", "OBJROOT=build",
+               "build"
+
+    (libexec/"lib").install Dir["External/build/macOS/lib{SPIRVCross,SPIRVTools,glslang}.a"]
+    glslang_dir = Pathname.new("External/glslang")
+    Pathname.glob("External/glslang/{glslang,SPIRV}/**/*.{h,hpp}") do |header|
+      header.chmod 0644
+      (libexec/"include"/header.parent.relative_path_from(glslang_dir)).install header
+    end
+    (libexec/"include").install "External/SPIRV-Cross/include/spirv_cross"
+    (libexec/"include").install "External/glslang/External/spirv-tools/include/spirv-tools"
+
+    frameworks.install "Package/Release/MoltenVK/macOS/framework/MoltenVK.framework"
+    lib.install "Package/Release/MoltenVK/macOS/dynamic/libMoltenVK.dylib"
+    lib.install "Package/Release/MoltenVK/macOS/static/libMoltenVK.a"
+    include.install "MoltenVK/MoltenVK/API" => "MoltenVK"
+    include.install "External/Vulkan-Headers/include/vulkan" => "vulkan"
+    include.install "External/Vulkan-Portability/include/vulkan" => "vulkan-portability"
+
+    bin.install "Package/Release/MoltenVKShaderConverter/Tools/MoltenVKShaderConverter"
```

Comment:
I think this might need an alias?

Reply:
I've got no idea about alias here, could you suggest one? Thanks.

Reply:
Just the name of the binary as the alias? I think it would recreate the entire structure in bin otherwise.

Reply:
Do you mean recreate in `/usr/local/bin`? I don't think it is the case, since most of binaries in `/usr/local/bin` is symbolic link to `/usr/local/Cellar/<package>/<version>/bin` no matter it is a symbolic link or binaries inside Cellars.
```
$ ls -l /usr/local/bin/MoltenVKShaderConverter 
lrwxr-xr-x  1 user  admin  54 Apr 26 13:21 /usr/local/bin/MoltenVKShaderConverter -> ../Cellar/molten-vk/1.0.34/bin/MoltenVKShaderConverter
```

Reply:
My mistake, brew is smarter then I thought. I expected it to become `/usr/local/bin/Package/Release/MoltenVKShaderConverter/Tools/MoltenVKShaderConverter`